simulator/simulator.py

```diff
@@ -0,0 +1,104 @@
+import datetime
+import json
+import secrets
+import time
+from collections.abc import Iterator
+from pprint import pprint
+from random import SystemRandom
+from typing import Final
+
+from firebase_admin import credentials, db, initialize_app
+from firebase_admin.credentials import Certificate
+from firebase_admin.db import Reference
+
+from config import \
+    COORDINATES_PATH, \
+    DATABASE_REFERENCE_PATH, \
+    DATABASE_URL, \
+    KEY_PATH, \
+    ULTRAVIOLET_VALUE_RANGE, \
+    AIR_TEMPERATURE_RANGE, \
+    WATER_TEMPERATURE_RANGE, \
+    WATER_DEPTH_RANGE, \
+    WIND_DIRECTION_RANGE, \
+    WIND_SPEED_RANGE, \
+    EXPECTED_GENERATED_ENV_DATA_AMOUNT
+from models import Coordinate, Temperature, Ultraviolet, WaterDepth, Wind, EnvData, CoordinateRaw
+
+
+def read_raw_coordinates_data() -> CoordinateRaw:
+    with open(COORDINATES_PATH) as json_file:
+        raw_coordinates_data: Final[CoordinateRaw] = json.load(json_file)
+
+    return raw_coordinates_data
+
+
+def insert_data(data: EnvData) -> None:
+    ref: Final[Reference] = db.reference(DATABASE_REFERENCE_PATH)
+    ref.set(data.to_json_serializable)
```

Comment:
@ricky9667 @xiao0818 Should we use `set` instead of `add` ?
Not sure the origin design of this step. Is it expect for "update the existing data on the cloud" or "add a random mock data to the cloud"?

Reply:
Issue opened in #15.